apps/opencs/model/tools/referenceablecheck.cpp

```diff
@@ -397,6 +397,41 @@ void CSMTools::ReferenceableCheckStage::containerCheck(
     //checking for name
     if (container.mName.empty())
         messages.push_back (std::make_pair (id, container.mId + " has an empty name"));
+    
+    //checking contained items
+    const std::vector<ESM::ContItem>& itemList = container.mInventory.mList;
+    for (unsigned i = 0; i < itemList.size(); ++i)
```

Comment:
Had to look up if 'unsigned i' is actually legal C++. Apparently it is. Can we change to 'unsigned int i' anyway just for clarity's sake?